File: codes/cpp/chap06/answer_A36.cpp
#include <iostream>
using namespace std;

int main() {
	int N, K;
	cin >> N >> K;
	if (K >= 2 * N - 2 && K % 2 == 0) cout << "Yes" << endl;
	else cout << "No" << endl;
	return 0;
}

File: codes/java/chap06/answer_A36.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		// 入力
		int N = sc.nextInt();
		int K = sc.nextInt();

		// 出力
		if (K >= 2*N-2 && K%2 == 0) System.out.println("Yes");
		else System.out.println("No");
	}
};
